cf/tasks/package-bits.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -x
+set -o errexit
+set -o nounset
+
+#find . -print
+
+# Determine configuration.
+
+VERSION=$(cat semver.scf-version/version)
+PLATFORM=$(echo $(basename s3.certstrap-binary/certstrap-*.tgz .tgz)|sed -e 's|\(.*\)\.\([^.]*\)$|\2|')
```

Comment:
This should just be passed in as an environment variable from the pipeline definition.